### PR TITLE
react-hook-form

### PR DESCRIPTION
**Pull request by svetlana-suslova**
Used react-hook-form in SaveRecord and SaveCategory.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "@types/react-redux": "7.1.9",
     "@types/react-router-dom": "5.1.5",
     "@types/toastr": "2.1.38",
-    "axios": "0.20.0",
+    "axios": "^0.21.1",
```
Comment (erik-sytnyk) [inline package.json]: Use exact versions of npm packages

https://medium.com/front-end-weekly/how-to-save-exact-npm-package-versions-7f8ff9a0415a

File: src/components/auth/LoginPage.tsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ import {AppState} from 'reducers';
 import validationHelper from 'helpers/validationHelper';
 
 import AppIcon from 'components/common/AppIcon';
-import TextInput from 'components/common/TextInput';
+import {TextInput} from 'components/common/TextInput';
```
Comment (erik-sytnyk) [inline src/components/auth/LoginPage.tsx:13]: Please, apply default prettier formatting

File: package.json (hunk 1 above)
Comment (svetlana-suslova) [reply package.json]: added .npmrc with save-exact=true to the project

File: src/components/auth/LoginPage.tsx (hunk 2 above)
Comment (svetlana-suslova) [reply src/components/auth/LoginPage.tsx:13]: applied

File: package.json (hunk 1 above)
Comment (svetlana-suslova) [reply package.json]: and changed package.json

**Pull request closed without merging**